htdocs/accountancy/journal/expensereportsjournal.php

```diff
@@ -214,8 +214,8 @@
 					$bookkeeping->label_operation = $tabuser[$key]['name'];
 					$bookkeeping->montant = $mt;
 					$bookkeeping->sens = ($mt >= 0) ? 'C' : 'D';
-					$bookkeeping->debit = ($mt <= 0) ? -$mt : 0;
-					$bookkeeping->credit = ($mt > 0) ? $mt : 0;
+					$bookkeeping->debit = ($mt <= 0) ? abs($mt) : 0;
+					$bookkeeping->credit = ($mt > 0) ? abs($mt) : 0;
```

Comment:
If amount is > 0, why using abs function. We can just keep $mt as it is like in current code ?

Reply:
Yes it work. We can keep it when $mt > 0, without abs().